_content/papers/sakkayaditthi_analayo.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Sakkayaditthi"
+authors:
+ - analayo
+editor: "W. G. Weeraratne"
+booktitle: "Encyclopaedia of Buddhism"
+volume: 7
+number: 4
+pages: "641--644"
+publisher: "The Department of Buddhist Affairs"
+address: "Sri Lanka"
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/encyclopedia-entries/sakkayaditthi.pdf"
+year: 2006
+drive_links:
+ - "https://drive.google.com/file/d/1CEpJAWaw7TX8YkdljyRjfFgRGCCtmQQY/view?usp=drive_link"
+course: view
+tags:
```

Comment:
`origination` is a bit redundant with `view` and [`inner`](https://buddhistuniversity.net/tags/inner) is the "self" tag.

```yaml
tags:
  - phenomenology
  - karma
  - inner
```